samples/sensor/bh1750/src/main.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2022 morsisko
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <devicetree.h>
+#include <drivers/sensor.h>
+
+/*
+ * Get a device structure from a devicetree node with compatible
+ * "rohm,bh1750". (If there are multiple, just pick one.)
+ */
+static const struct device *get_bh1750_device(void)
+{
+	const struct device *dev = DEVICE_DT_GET_ANY(rohm_bh1750);
```

Comment:
Use `DEVICE_DT_GET_ONE` instead to fail at build time if there is no device available in the devicetree. You can then skip the NULL check below.